api/routers/cameras.py

```diff
@@ -143,6 +143,35 @@ async def get_camera(camera_id: str):
     return camera
 
 
+def get_first_unused_id(arr):
+    if not arr:
```

Comment:
I think that the `arr` parameter should be renamed as `cameras_ids` (or another descriptive name)
Also, I think that this logic can be simpler if we use regular expressions.
For example, something like:
```
if not cameras_ids:
    return 0
is_a_number = re.compile("^[0-9]+$")
number_ids = [int(id) for id in cameras_ids if is_a_number.match(id)]
if not number_ids:
    return 0
return max(number_ids) + 1
```

Reply:
I agree with most of the recommendations here, however, i want to highlight one problem. If a user manually sets the id=40, the code will suggest the next id to be 41, and maybe there are many id's available before 40.
Should i keep these part of the code:
```
ids_numbers.sort()
result = 0
for i in range(0, ids_numbers[len(ids_numbers)-1]+1):
    if ids_numbers[i] != i:
        result = i
        break
else:
    result = ids_numbers[len(ids_numbers)-1] + 1

return result
```

Reply:
Yes, it could work. Maybe you can remove the `break` using instead a return.
Something like:
```
for i in range(0, ids_numbers[len(ids_numbers)-1]+1):
    if ids_numbers[i] != i:
        return i
return ids_numbers[len(ids_numbers)-1] + 1
```

What do you think?

Reply:
Of course, I think it is a good idea. I will change it!